WIP: adding campaign support

mtam2 opened the pull request:
TODO:
- update documentation
- add tests

mtam2 committed:
Add campaign & campaignMessage support

rylwin commented on lib/active_campaign/api/campaign_messages.rb:
```diff
@@ -0,0 +1,23 @@
+    #
+    # Interface to campaign endpoints
+    #
+    # @author Mikael Henriksson <mikael@mhenrixon.com>
```
@mtam2 I think you get to be the author of these changes if you want

rylwin commented on lib/active_campaign/client.rb:
```diff
@@ -13,6 +13,8 @@ class Client
     endpoint :accounts
     endpoint :addresses
+    endpoint :campaigns
+    endpoint :campaign_messages
```
swap the order of the new lines to maintain alpha order?

rylwin reviewed (approved):
Looks great! 2 minor comments for your consideration. :rocket: when ready!

mtam2 commented:
Awesome, I've reordered the endpoints, changed the author to me, and rebased this into 1 commit. Only you have permission to merge so I'll update the Gemfile after the merge.
